### PR TITLE
Monitoring

### PR DESCRIPTION
**Pull request by michalc**

### CODE REVIEW

File: core/app/app_outgoing.py [hunk 1]
```diff
@@ -214,7 +177,8 @@ def feed_unique_ids(feed_endpoints):
         while href:
             updates_href = href
             href = await ingest_feed_page(
-                logger, metrics, session, 'full', feed_lock, feed, es_endpoint, [index_name], href
+                logger, metrics, redis_client, session, 'full', feed_lock, feed, es_endpoint, [
```
Comment (lgarvey) [inline core/app/app_outgoing.py]: Just a thought: a lot of funcs have a large number of args, several of which appear to be essentially cross cutting concerns - redis client, sentry, logging, metrics etc.  It'd be less explicit, but I wonder if it'd be worth collapsing those args into a single `context` arg?  The remaining function signature would better document its purpose/single-responsibility.

File: core/app/app_server.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
 from aiohttp import web
 import mohawk
 from mohawk.exc import HawkFail
 
 from shared.logger import (
+    logged,
     get_child_logger,
 )
 from shared.utils import (
```
Comment (lgarvey) [inline core/app/app_server.py]: Could be worth putting the es_* stuff into an app_es module, similar to app_redis.  SRP at the module level.

Comment (michalc) [reply core/app/app_server.py]: Ah most of it already is! In `app_elasticsearch`. (The one possible exception is that there is some AWS signing code in shared.utils, which is only used for elasticsearch, but it's also used by the elasticsearch proxy)

File: core/app/app_outgoing.py (hunk 1 above)
Comment (michalc) [reply core/app/app_outgoing.py]: I think this sounds very reasonable. I might make a separate refactor PR for this though.

**Pull request merged**